db/post_migrate/20220214224506_reset_custom_emoji_post_bakes_version_secure_fix.rb

```diff
@@ -2,7 +2,9 @@
 
 class ResetCustomEmojiPostBakesVersionSecureFix < ActiveRecord::Migration[6.1]
   def up
-    if SiteSetting.secure_media
+    secure_media_enabled = execute("SELECT value FROM site_settings WHERE name = 'secure_media'")
```

Comment:
Hmm can't we just use `DB.query_single` and avoid this ntuples wrangling?

Reply:
This is used in plenty of other post_migrate code

Reply:
Good point :) I've updated the PR to reflect that.